src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,932 @@
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	SET default_transaction_isolation = 'read committed';
+	~~~
+
+- The `default_transaction_isolation` session variable as a [connection parameter]({% link {{ page.version.version }}/connection-parameters.md %}#connect-using-a-url) with [`cockroach sql`]({% link {{ page.version.version }}/cockroach-sql.md %}):
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	cockroach sql -–url='postgresql://{username}@{host}:{port}/{database}?options=-c default_transaction_isolation=read%20committed'
+	~~~
+
+To view the isolation level of a session:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW default_transaction_isolation;
+~~~
+
+~~~
+	default_transaction_isolation
+-----------------------------------------------------
+	read committed
+~~~
+
+### Set the current transaction to `READ COMMITTED`
+
+To begin a transaction as a `READ COMMITTED` transaction, use one of the following options:
+
+- The [`BEGIN TRANSACTION ISOLATION LEVEL`]({% link {{ page.version.version }}/begin-transaction.md %}#parameters) statement:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
+	~~~
+
+- The [`SET TRANSACTION ISOLATION LEVEL`]({% link {{ page.version.version }}/set-transaction.md %}#parameters) statement, at the beginning of the transaction:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	BEGIN;
+	  SET TRANSACTION ISOLATION LEVEL READ COMMITTED;
+	~~~
+
+- The `transaction_isolation` [session variable]({% link {{ page.version.version }}/session-variables.md %}), at the beginning of the transaction: 
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	BEGIN;
+	  SET transaction_isolation = 'read committed';
+	~~~
+
+To view the isolation level of a transaction, run `SHOW` within the open transaction:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW transaction_isolation;
+~~~
+
+~~~
+  transaction_isolation
+-------------------------
+  read committed
+~~~
+
+{{site.data.alerts.callout_success}}
+Starting a transaction as `READ COMMITTED` does not affect the [session-level isolation setting](#set-the-current-session-to-read-committed), which can be different.
+{{site.data.alerts.end}}
+
+## `READ COMMITTED` transaction behavior
+
+`READ COMMITTED` and `SERIALIZABLE` transactions both serve globally consistent ("non-stale") reads and [commit atomically]({% link {{ page.version.version }}/developer-basics.md %}#how-transactions-work-in-cockroachdb). `READ COMMITTED` transactions have the following differences:
+
+- Writes in concurrent `READ COMMITTED` transactions can interleave without blocking or aborting transactions, and a write can never block a read of the same row. This is because `READ COMMITTED` transactions are not required to be placed into a [serializable ordering]({% link {{ page.version.version }}/demo-serializable.md %}). 
```

Comment:
I think this is what I meant, since lost update anomalies seem to be caused by interleaving writes that don't prevent a txn from committing:

> Writes in concurrent `READ COMMITTED` transactions can interleave without aborting transactions, and a write can never block a non-locking read of the same row.

Is that correct? @nvanbenschoten